client/src/modules/roles/modal/userRole.js

```diff
@@ -18,6 +18,9 @@ function UsersRolesController(data, $uibModal, $uibModalInstance, RolesService,
     RolesService.userRoles(vm.user.id)
       .then(roles => {
         vm.roles = roles;
+        vm.selected = roles
+          .filter(role => role.affected === 1)
```

Comment:
Just a note for a future date; nothing to do with this PR.

We have role "assignments", but sometimes we use the word "affected".  This is a mix of French and English.  We might want to change the names of "affected" variables to "assigned".